src/python/WMComponent/DBS3Buffer/DBSUploadPoller.py

```diff
@@ -293,6 +305,9 @@ def algorithm(self, parameters=None):
         """
         try:
             logging.info("Starting the DBSUpload Polling Cycle")
+            # refreshing parentageCache every cycle
+            self.updateDatasetParentageCache()
+            logging.info("Dataset parentage map: %s" % self.datasetParentageCache)
```

Comment:
This log line can be very very large. Maybe we should convert it to debug or maybe only print the number of parent/child dataset maps.

Reply:
It was just meant for testing. I will make to debug for now.  The number of parent/child give much information